nucliadb_texts/src/writer.rs

```diff
@@ -201,6 +194,24 @@ impl TextWriterService {
         })
     }
 
+    fn try_create_index_dir(path: &Path) -> NodeResult<()> {
+        let result = fs::create_dir(path);
+        if let Err(error) = result {
+            if path.exists() {
+                // operation failed but directory exists, we must delete it
+                if let Err(remove_error) = std::fs::remove_dir(path) {
+                    return Err(node_error!(
+                        "Double error creating and removing texts directory: \nFirst: {error} \
```

Comment:
is the extra `\` intended?

Reply:
Yes, it comes from `rustfmt` splitting the string